static/frontEndLayout.js

```diff
@@ -168,19 +173,19 @@ document.onreadystatechange = function() {
 };
 
 var buy = function(seed) {
```

Comment:
one change now is that when you buy, you're buying with a recipe_id instead of a seed id.  It's a small thing, but changing the name of this parameter will make it easier to understand in the future.